SpatioTemporalForecaster/spatiotemporalforecaster/dataset.py

```diff
@@ -0,0 +1,119 @@
+"""
+
+"""
+import torch
+import pandas as pd
+import numpy as np
+
+def set_torch_tensor(df):
```

Comment:
This and `ensure_dimensions_match` feel like they should be static methods of Dataset